scripts/rename_session_windows.py

```diff
@@ -23,6 +24,32 @@ def get_option(server: libtmux.Server, option: str, default: Any) -> Any:
 
     return eval(out[0])
 
+
+def set_option(server: libtmux.Server, option: str, val: str):
+    server.cmd('set-option', '-g', f'{OPTIONS_PREFIX}{option}', val)
+
+
+def get_window_option(server: libtmux.Server, window_id: str|None, option:str, default:Any) -> Any:
```

Comment:
I want to keep the python requirement as low as I can, please switch the `str|None` to `Optional[str]`